app/Http/Controllers/Admin/BatchController.php

```diff
@@ -41,21 +45,35 @@ public function edit($id)
         $programs = $this->programRepository->listprogramArray();
         $positions = $this->positionRepository->listpositionArray();
         $workspaces = $this->workspaceRepository->listWorkspaceArray();
-        $batch = $this->batch->with('program', 'position', 'workspace')->find($id);
+        $batch = $this->batch->with('program', 'position', 'workspace', 'subjects')->find($id);
+        $subjects = $this->subject->getNameSubject()->pluck('name', 'id');
 
-        return view('admin.batches.edit', compact('batch', 'programs', 'positions', 'workspaces'));
+        return view('admin.batches.edit', compact('batch', 'programs', 'positions', 'workspaces', 'subjects'));
     }
 
     public function update(BatchRequest $request, $id)
     {
-        $this->batch->update($request->only([
-            'stop_day',
-            'start_day',
-            'position_id',
-            'program_id',
-            'workspace_id',
-            'batch',
-        ]), $id);
+        DB::beginTransaction();
+
+        try {
+            $this->batch->update($request->only([
+                'stop_day',
+                'start_day',
+                'position_id',
+                'program_id',
+                'workspace_id'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/BatchController.php</br>**Line**: 64</br>Comma required after last value in array declaration